### PR TITLE
refactor(multiple): clean up IE vendor prefixes

### PR DESCRIPTION
**Pull request by crisbeto**
Cleans up the styles specifically for IE and Edge.

### CODE REVIEW

**Review by devversion — APPROVED** [review]

**Review by jelbourn — APPROVED** [review]
LGTM

File: src/material/core/style/_vendor-prefixes.scss [hunk 1]
```diff
@@ -18,10 +17,6 @@
   &::-webkit-input-placeholder {
     @content;
   }
-
-  &:-ms-input-placeholder {
```
Comment (mmalerba) [inline src/material/core/style/_vendor-prefixes.scss:22]: Somehow this change is causing a screenshot diff in Chrome, can we move it to a separate PR to investigate so we can merge the rest of this PR?

**Review by mmalerba — APPROVED** [review]

File: src/material/core/style/_vendor-prefixes.scss (hunk 1 above)
Comment (crisbeto) [reply src/material/core/style/_vendor-prefixes.scss:22]: Done.

**Comment by mmalerba** [comment]
needs rebase

**Comment by crisbeto** [comment]
Rebased.
